### DISCUSSION TITLE
Difficulty in installation

### DISCUSSION THREAD

**Discussion by unicorn-23** [Starlitnightly/omicverse#161] [General]
It should be clearly specified in the requirements that the corresponding version of the third-party library should be set. After trying for a long time, it was found that the scanpy version needs to be set to 1.10.1

**Comment by Starlitnightly** [comment]
Thank you for your comments, we hope that scanpy greater than 1.8 are compatible, so if you encountered the version report error, welcome to submit an issue to us, we will promptly fix the version conflict error.
